packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -234,6 +234,16 @@ export const InputPrompt: React.FC<InputPromptProps> = ({
         buffer.moveToOffset(cpLen(buffer.text));
         return;
       }
+      // Ctrl+C (Clear input)
+      if (key.ctrl && (input === 'c')) {
+        if (buffer.text.length > 0) {
+          buffer.setText('');
+          resetCompletionState();
+          return;
+        }
+        // If buffer is empty, let the global handler deal with exit flow
```

Comment:
Nit: can you  remove this comment as it is a little misleading as it isnt like the keyboard handlers are coupled together.